microsoft-edge/devtools-guide-chromium/javascript/ado-symbol-server.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Debug original code using the Azure Artifacts symbol server
+description: Learn how to publish source maps to the Azure Artifacts symbol server from Azure DevOps to debug original source code in DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/24/2022
+---
+
+# Debug original code using the Azure Artifacts symbol server
```

Comment:
I'd recommend keeping this section and expanding it in this doc to show the value prop of this feature.  Maybe we can show how debugging this demo site: https://microsoftedge.github.io/Demos/devtools-a11y-testing/ or another demo for this feature is hard with minification, transpilation, etc.

I'd break out the other 2 sections into 2 separate docs and then have a section like:
## Get started
1. [Publish symbols for debugging](/microsoft-edge/devtools-guide-chromium/javascript/publish-sourcemaps.md) 
and this can link to the Azure DevOps task that you have but we also need the content about computing the SHA-256 hash of your script. I've attached Rob's draft Markdown here for additonal context: 
[index.md](https://github.com/MicrosoftDocs/edge-developer/files/8157998/index.md)
2. [Fetch sourcemaps from Azure Artifacts Symbol Server in DevTools](/microsoft-edge/devtools-guide-chromium/javascript/fetch-sourcemaps.md)

The way I think about is customers who want to have this feature need to do 2 things; 
1. Publish sourcemaps to ADO
2. Fetch sourcemaps from ADO in DevTools
and the customers may be responsible for both things or they may be mutually exclusive.

Reply:
Let's see if I'm understanding this well. 3 docs:

1. What's source maps, showing how hard it can be do debug in production
2. How to publish symbols
3. How to fetch source maps from devtools

What do you think about it @mikehoffms ?
My impression is that the first article is likely not needed in that case. If it's only background info about why source maps are important, then I'd rather keep it as a short section in one of the 2 other articles (or both). Or, maybe we keep it, but use it to link to the 2 other articles.

As for splitting in 2 articles or not, I thought about it at first, but then decided to keep them in 1 article because they were so related. Yes, the steps may be followed by 2 different people, but I don't think there is a case where you would do one and not the other, right? Because of this, it seemed like merging them in one article made more sense. 

I'd like Mike's point of view on this too.

Reply:
After reading Rob's index.md, it is clear that there is much more content to be added to step 2 which I was not aware of.
Because of this, it does seem like having it as a separate article makes more sense.

Reply:
Ok, I just pushed a couple of commits to address this. The article has now been split into 2 (1. publish, 2. download). 
All 3 articles (about source maps, publish, download) now introduce what source maps are and why they are useful to website authors, and cross-link appropriately.

@mikehoffms I also simplified the "about source maps" article and removed 2 things:

- `//@ sourceURL` pragma has not been supported for a really long time, no need to confuse readers by even mentioning it
- the demo showing how to use it. It made up a big part of the article, but wasn't the most important part of it. And I don't think people, nowadays, rely as much on evaluating their files themselves as they use to. I simply don't think the demo adds value to the article.